src/Compilers/CSharp/Portable/FlowAnalysis/DataFlowsInWalker.cs

```diff
@@ -91,15 +91,20 @@ public override BoundNode VisitRangeVariable(BoundRangeVariable node)
             return null;
         }
 
-        protected override void ReportUnassigned(Symbol symbol, SyntaxNode node)
+        protected override void ReportUnassigned(Symbol symbol, SyntaxNode node, int slot, bool skipIfUseBeforeDeclaration)
         {
             // TODO: how to handle fields of structs?
```

Comment:
It features no more specific handling than it did before. So I would say it handles no more than it did before, but I don't know what this comment meant anyway -- it appears to me that this handles struct fields just fine.